### DISCUSSION TITLE
Touch Events Fixed

### DISCUSSION THREAD

**Discussion by rabbibotton** [rabbibotton/clog#322] [General]
I have redone Tutorial 8 and fixed the Touch based events making working with Touch events on iOS and Android easier and to better handle mouse events on mobile.

I have some more things to clean up and fix up and then will do another release of CLOG. My push for now will be ease of development for Android and hopefully iOS and then work towards a few new ideas for CLOG I will discuss soon.

**Comment by rabbibotton** [comment]
I have further cleaned up tutorial 8 as it ends up as long as the on-touch-start event is set then the on-pointer events all work as expected. Much of the builder now is working on mobile devices as well. I have a bit more work to do to convert all use of on-mouse events to on-pointer events and to set on-touch-start events to support touch.

**Comment by aykaramba** [comment]
Oh wow. Touch (drag) works on Android. Nice! Very nice. Thank you kindly.
